### PR TITLE
Add minder OSPS minder profiles and first rules

### PR DESCRIPTION
**Pull request by puerco**
This pull request adds the three skeletons for minder profiles for each level of the baseline, the are written to be applied incrementally as a project matures:

- `osps-baseline-level1.yaml` → Level 1
- `osps-baseline-level2.yaml` → Level 2
- `osps-baseline-level3.yaml` → Level 3

This PR is intended to add the initial template for the profiles, but it already includes a number of rule types that can check for the following controls:

```
OSPS-AC-03
OSPS-AC-04
OSPS-BR-06
OSPS-DO-02
OSPS-DO-03
OSPS-DO-04
OSPS-QA-04
OSPS-QA-06
```

Signed-off-by: Adolfo García Veytia (puerco) <puerco@stacklok.com>

### CODE REVIEW

**Review by SecurityCRob — APPROVED** [review]

File: resources/minder/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# OSPS Minder Profiles
+
+[Minder](https://github.com/mindersec/minder) is an Open Source Supply Chain
+Security plaform part of the OpenSSF ecosystem. A Minder profile is a security
+policy that groups together rules that are constantly evaluating against entities
+in your software project.
+
+This directory contains three [Minder](https://github.com/mindersec/minder) 
+profiles that any project can use to monitor its compliance with the
```
Comment (david-a-wheeler) [inline resources/minder/README.md:9]:
```suggestion
profiles that any project can use to experimentally monitor its compliance with the 
```

File: resources/minder/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# OSPS Minder Profiles
+
+[Minder](https://github.com/mindersec/minder) is an Open Source Supply Chain
+Security plaform part of the OpenSSF ecosystem. A Minder profile is a security
+policy that groups together rules that are constantly evaluating against entities
+in your software project.
+
+This directory contains three [Minder](https://github.com/mindersec/minder) 
+profiles that any project can use to monitor its compliance with the 
+Open Source Project Security Baseline.
```
Comment (david-a-wheeler) [inline resources/minder/README.md:10]:
```suggestion
Open Source Project Security Baseline.
Note that the baseline is still in development, as are these profiles.
```

**Comment by david-a-wheeler** [comment]
Since we're still working on some things, I think it's really important to be up-front that this is early-stage work. I *applaud* early-stage work, don't get me wrong, I just don't want people to grab this & think it's all done now. :-).

I made a few tweaks above. As long as we mark things like that (it could be noted differently), so people understand its early nature, then wonderful!

**Review by eddie-knight — CHANGES_REQUESTED** [review]
Agree with @david-a-wheeler, and I think we should stabilize this before merging it in

**Comment by puerco** [comment]
Thanks for the review @david-a-wheeler, I also agree not to merge it until done. I'll close it as I won't be working on it anymore.

**Pull request closed without merging**